engine/src/main/java/org/terasology/rendering/dag/nodes/OutputToScreenNode.java

```diff
@@ -59,4 +80,35 @@ public void process() {
         renderFullscreenQuad();
         PerformanceMonitor.endActivity();
     }
+
+    public static OutputToScreenNode getInstance() {
+        return nodeInstance;
```

Comment:
Definitely do not like this at this stage. There could be multiple Output nodes in a graph, with only one enabled at any given time. It might be meaningful at some point to impose such limitation, but I wouldn't do I just yet.